include/evmc/evmc.hpp

```diff
@@ -127,6 +127,21 @@ class vm
         return result{m_instance->execute(m_instance, &ctx, rev, &msg, code, code_size)};
     }
 
+    /// Executes code without the Host context.
+    ///
+    /// The same as
+    /// execute(evmc_context&, evmc_revision, const evmc_message&, const uint8_t*, size_t),
+    /// but without providing the Host context.
+    /// This method is for experimental precompiles support where execution is guaranteed
+    /// not to require any Host access.
+    result execute(evmc_revision rev,
```

Comment:
Can we mark this experimental, e.g. this may be removed across non-breaking versions?

Reply:
We don't need to merge it yet.